workers/unity/Assets/Gdk/Core/Tests/Editmode/Components/OptionTests.cs

```diff
@@ -0,0 +1,179 @@
+using System;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.Core.EditmodeTests
+{
+    [TestFixture]
+    public class OptionTests
+    {
+        [Test]
+        public void Parameterless_constructor_creates_empty_option()
+        {
+            var option = new Option<bool>();
+            Assert.AreEqual(false, (bool)option.HasValue);
```

Comment:
this looks like it needs linting

Reply:
Yep, haven't run the linter yet. Will do.